packages/oas-to-har/package.json

```diff
@@ -39,7 +39,7 @@
     "url": "https://github.com/readmeio/oas/issues"
   },
   "scripts": {
-    "attw": "attw --pack --format table-flipped",
+    "attw": "attw --pack --format table-flipped --profile node16",
```

Comment:
i'd probably remove these `--format` flags and defer to their auto-formatting, the output in github actions looks kinda screwy (and it looks even worse on narrower screens):

![CleanShot 2025-03-04 at 14 14 03@2x](https://github.com/user-attachments/assets/1a7d9b45-2e49-4b02-bb8e-a25253c52b63)


```suggestion
    "attw": "attw --pack --profile node16",
```

Reply:
changing this to `ascii` because they default to `auto` which still ends up using the table format on libraries that just have a single or two exports. makes the whole output look inconsistent with some having a table and others being a raw dump.